app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,83 @@
+class TweetsController < ApplicationController
+
+  get '/tweets' do
+    if session[:user_id]
+      @user = User.find_by_id(session[:user_id])
+      @tweets = Tweet.all
+      erb :'tweets/tweets'
+    else
+      redirect to '/login'
+    end
+  end
+
+  get '/tweets/new' do
+    if session[:user_id]
+      erb :'tweets/create_tweet'
+    else
+      redirect to '/login'
+    end
+  end
+
+  post '/tweets/new' do
+    if user = User.find_by_id(session[:user_id])
+      if user.tweets << Tweet.new(content: params[:content])
```

Comment:
The tweet creation and user association can both be accomplished in the same step:

``` ruby
@tweet = Tweet.create(content: params[:content], user_id: user.id)
```